e2e/keywords/utility.resource

```diff
@@ -0,0 +1,28 @@
+*** Settings ***
+Documentation     
+...               Keywords defined here can be used where this Keywords.resource in loaded.
+Library    ../libs/steps.py
+
+*** Variable ***
+${retry_time_out}=    600
+${retry_interval}=    5
+
+*** Keywords ***
+Check volume state should be ${volume_expect_state}
+    ${volume_current_state} =   get_volume_state   ${volume_name}
+    Should Be Equal    ${volume_current_state}    ${volume_expect_state}
+
+Check node ${node_number} state should be ${node_expect_state}
```

Comment:
```suggestion
Node ${node_number} state should be ${expected_node_state}
```

Reply:
Suggest moving to a node.resource.

Reply:
Will update the description and move it into `node.resource`. Thank you!